content/docs/reference/cookies.mdx

```diff
@@ -161,6 +161,12 @@ cookie:
 
 ## Cookie Secure {#cookie-secure}
 
+:::caution
+
+As of v0.25.0, we have deprecated the **Cookie Secure** setting in Pomerium Core, the Enterprise Console, and the Ingress Controller and will remove it in **v0.26.0**. It will be enabled by default going forward.
```

Comment:
Do you think it helps to mention all of Core, Enterprise Console, and Ingress Controller?

I wonder if it would be simpler just to say: "As of v0.25, the **Cookie Secure** setting is deprecated."